unicodetools/src/main/java/org/unicode/tools/emoji/Emoji.java

```diff
@@ -103,62 +102,49 @@ public class Emoji {
     public static final VersionInfo VERSION12_1 = VersionInfo.getInstance(12, 1);
     public static final VersionInfo VERSION12 = VersionInfo.getInstance(12);
     public static final VersionInfo VERSION11 = VersionInfo.getInstance(11);
+    public static final VersionInfo VERSION10 = VersionInfo.getInstance(10);
```

Comment:
What are these for? They don’t correspond to any emoji versions I’m aware of.

Reply:
Rather than duplicating some VersionInfo instances in both VERSIONxx and UCDxx I made everything just VERSIONxx. There were some UCDxx that didn't already have VERSIONxx. Having just one kind of constant for a VersionInfo constant seems much simpler. The VersionInfo itself doesn't know whether it's for UCD or emoji.

These are used in EMOJI_TO_UNICODE_VERSION, on the right side.